packages/api/src/repository/Feed.ts

```diff
@@ -46,11 +46,21 @@ export class FeedRepository {
 
   async getFeeds (
     page: number,
-    size: number
+    size: number,
+    network: string
   ): Promise<Array<FeedDbObjectNormalized>> {
+    let query
+    network !== 'all'
+      ? (query = this.collection.find({
+          feedFullName: { $in: this.dataFeedsFullNames },
+          network
+        }))
+      : (query = this.collection.find({
+          feedFullName: { $in: this.dataFeedsFullNames }
+        }))
```

Comment:
As a rule of thumb, ternaries shouldn't be used as a replacement for `if`/`else` statements, and they shouldn't have procedural code in them (e.g. variable binding / value assignment). They're better suited for expressions:

```suggestion
    // Base query will filter by allowed feed names.
    // If a network is specified, the query will be refined to only include feeds for that network. 
    const baseQuery = { feedFullName: { $in: this.dataFeedsFullNames } }
    const query = network === 'all' ? baseQuery : { ...baseQuery, network }
```

Reply:
Or for a mutable, simpler approach:

```suggestion
    // Base query will filter by allowed feed names.
    // If a network is specified, the query will be refined to only include feeds for that network. 
    const query = { feedFullName: { $in: this.dataFeedsFullNames } }
    if (network !== 'all') {
      query['network'] = network
    }
```